### DISCUSSION TITLE
Lodestar Standup 2024-15 (April 16, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6671] [Standups]
### Meeting Info
[Apr 16, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/apr-16-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.18.0-rc.0 review
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by wemeetagain** [comment]
I'd like to discuss hosting a "pair-programming" / collaborative session on either verkle or peerDAS this week (Thursday ?) if anyone else is interested / available. And do more sessions in coming weeks as others are available or willing.

The idea would be to try to come with at least a high level understanding of the feature, but then go thru the relevent EIPs / PRs / issues / posts / our own branches / etc. to better understand things in a few ways. And going thru things collaboratively can be helpful for answering latent questions and getting a better understanding individually.

- Understanding how the specification delivers the desired functionality, which design decisions were made, possible alternatives, where the "knobs" are, open questions, etc.
- Understanding the specification from an implementation perspective, what is required for the beacon node
- Digging into the state of our implementations, determining what work is outstanding
- Answering or at least documenting any questions that arise from digging
- Understanding the state of current discussions, opportunities for comment or collaboration

**Reply by philknows** [reply]
We have some of these days internally scheduled as "Ethereum Guild" meetings where we'll go through it together in depth. Currently scheduled for April 25 and May 2. If you think we should have more sessions, we can discuss on the call.

**Reply by twoeths** [reply]
I guess it's almost impossible to have everyone understand everything thoroughly for new specs, to best prepare for the interop we need to come up with a plan. I propose we need at least 2 persons to study/take charge of each of the spec:
 - PeerDAS
 - Verkle Tree
 - EIP-6110
 - EIP-7002
 - EIP-7251
 - EIP-7549

one can pick multiples or all if he wants 😄

**Reply by nflaig** [reply]
Most of the EIP related work has been done by @g11tech and @ensi321 so far, I like the idea to have at least 2 people on each EIP, the second person might just be a reviewer / "co-author" but it's good idea to have more eyes on this as early as possible.

**Comment by twoeths** [comment]
Node 16 failed everywhere in ssz for example this PR https://github.com/ChainSafe/ssz/pull/366
should we support it anymore?

**Reply by nflaig** [reply]
Node 16 is [no longer maintained](https://github.com/nodejs/Release?tab=readme-ov-file#release-schedule) and has known security vulnerabilities. We should drop it, and might even do it without a breaking change, there is no reason for people to run this anymore.

**Comment by philknows** [comment]
Closing for #6678
